docs/intro.rst

```diff
@@ -179,7 +179,7 @@ generating
 Requirements
 ------------
 
-- Kafka: 0.9.0.1 (0.9.0.1-cp1 recommended)
+- Kafka: 0.10.0.0-SNAPSHOT (0.10.0.0-SNAPSHOT recommended)
```

Comment:
Yeah, the intent here was just to keep the format so when we update to 0.10.0.0, possibly w/ a `-cpX` postfix, we'll make sure to emphasize what we recommend.